PreviousComapnyName.txt

```diff
@@ -1 +1,5 @@
+My Previous company is Capgemini.
+hello
+
```

Comment:
remove unnecessary spaces

Reply:
Removed spaces, can you please review it again